FULL_HELP_DOCS.md

```diff
@@ -1212,6 +1212,7 @@ Configure connection to networks
 * `add` — Add a new network
 * `rm` — Remove a network
 * `ls` — List networks
+* `config-settings` — Fetch the config settings for the network
```

Comment:
can this be named `config` or `settings` instead?

Reply:
The reason I used the term `config-settings` is because that's what each of the config items are called. Each is defined as a "ConfigSettingId", and then a ConfigSettingEntry. I can reduce it to `config`, but that might be more ambiguous and folks might think it's doing something with local network config.

Maybe I shouldn't be bundling this under the network command.

I'm thinking I should move this under the ledger subcommand, since this is pulling information from the ledger.

Reply:
I've renamed it to `settings`, since the term settings seems to have the least overlap with other terms already used in the CLI for other things.